apps/beeai-ui/src/components/AnimateHeightChange/AnimateHeightChange.tsx

```diff
@@ -29,9 +29,10 @@ export function AnimateHeightChange({
   useEffect(() => {
     if (containerRef.current) {
       const resizeObserver = new ResizeObserver((entries) => {
-        const observedHeight = entries[0].contentRect.height;
-
-        setHeight(observedHeight);
+        if (entries.length > 0) {
+          const observedHeight = entries[0].contentRect.height;
+          setHeight(observedHeight);
+        }
```

Comment:
I'd prefer this approach:
```suggestion
        const observedElem = entries.at(0);
        if (observedElem) {
          setHeight(observedElem.contentRect.height);
        }
```